### PR TITLE
Update typography

### PR DESCRIPTION
**Pull request by daileytj**
Fixes PXBLUE-1230.

Changes proposed in this Pull Request:
- Update React Native theme to match DSM

### CODE REVIEW

**Comment by joebochill** [comment]
Code changes look reasonable to me. Can we see a before/after screenshot of what the showcase looks like to verify that the new text variants/styles look correct in our components?

**Comment by daileytj** [comment]
Pre-theme update 1
Pre theme update 1

Post-theme update 1
Post-theme update 1

Pre-theme update 2
pre theme update 2

Post-theme update 2
Post-theme update 2

File: components/src/core/score-card/score-card.tsx [hunk 1]
```diff
@@ -183,36 +183,36 @@ const HeaderText: React.FC<HeaderTextProps> = (props) => {
     const textColor = color || 'white';
     return (
         <View style={{ flex: 1 }}>
-            <Typography.H7
+            <Typography.H6
```
Comment (joebochill) [inline components/src/core/score-card/score-card.tsx:186]: the fontsize here in the scorecard needs to be updated to 18...H6 default is too large

Comment (daileytj) [reply components/src/core/score-card/score-card.tsx:186]: Is it ok to add a "mediumLarge" variant with a size of 18 to the sizes/index.ts file?

**Pull request merged**